src/main/java/nl/nn/testtool/Report.java

```diff
@@ -127,6 +133,14 @@ public class Report implements Serializable {
 	private transient boolean logMaxMemoryUsage = true;
 	private transient Map<Object, Set<Checkpoint>> streamingMessageListeners = new HashMap<>();
 	private transient Map<Object, StreamingMessageResult> streamingMessageResults = new HashMap<>();
+	private transient final OpenTelemetry openTelemetry;
+	private transient final Tracer tracer;
+
+	@Autowired
+	Report (OpenTelemetry openTelemetry) {
```

Comment:
Do we need an extra constructor? Report has a reference to TestTool (var testTool), wouldn't it be better to do testTool.getOpenTelemetry() when needed?

Reply:
I removed this constructor